cocos/particle/particle-utils.ts

```diff
@@ -21,7 +21,7 @@ export class ParticleUtils {
         }
         if (!this.particleSystemPool.has(prefab._uuid)) {
             this.particleSystemPool.set(prefab._uuid, new Pool<CCObject>(() => {
-                return instantiate(prefab);
+                return instantiate(prefab) || new Node();
```

Comment:
@caryliu1999 is this acceptable ? instantiate could fail

Reply:
> is this acceptable ? instantiate could fail

Yes, this method is not used currently.